server/src/test/integration/org/apache/cassandra/sidecar/testing/IntegrationTestModule.java

```diff
@@ -161,13 +212,6 @@ public Session getIfConnected()
         return cqlSessionProvider;
     }
 
-    @Provides
-    @Singleton
-    public ClusterLease clusterLease()
-    {
-        return new ClusterLease(ClusterLease.Ownership.CLAIMED);
-    }
-
```

Comment:
remove the fake lease. It now runs the actual lease competition in every integration test.